src/WebUI/Common/ResourceStatus.tsx

```diff
@@ -23,13 +24,15 @@ export class ResourceStatus extends BaseComponent<IResourceStatusProps, {}> {
                             {this._getStatus()}
                         </TooltipHost> : this._getStatus()
                 }
-                {
-                    this.props.statusDescription && 
-                    <span className="kube-status-desc">{this.props.statusDescription}</span>
-                }
-                {
-                    this.props.customDescription
-                }
+                <div className="description-padding">
```

Comment:
i am assuming you will fix TooltipHost to use tooltip.

Reply:
There is already a PR out which will fix this.